doc/hardware_acceleration.rst

```diff
@@ -0,0 +1,157 @@
+| SHACAL2   | NEON                                       | `shacal2_simd`     | 2.3.0      |
++-----------+--------------------------------------------+--------------------+------------+
+| SM4       | ARMv8 Cryptography Extensions (arm64 only) | `sm4_armv8`        | 2.8.0      |
++-----------+--------------------------------------------+--------------------+------------+
+| SHA-1     | ARMv8 Cryptography Extensions (arm64 only) | `sha1_armv8`       | 2.2.0      |
++-----------+--------------------------------------------+--------------------+------------+
+| SHA-256   | ARMv8 Cryptography Extensions (arm64 only) | `sha2_32_armv8`    | 2.2.0      |
++-----------+--------------------------------------------+--------------------+------------+
+
+PowerPC
+--------------
+
+On ppc64 and ppc32 platforms, the following CPU specific optimizations are available:
+
++-----------+--------------------------------------------+--------------------+------------+
+| Algorithm | Extension                                  | Module             | Added in   |
++===========+============================================+====================+============+
+| AES       | POWER8/POWER9                              | `aes_power8`       | 2.14.0     |
+|           |                                            |                    |            |
+|           | AltiVec                                    | `aes_vperm`        | 2.12.0     |
++-----------+--------------------------------------------+--------------------+------------+
+| AES-GCM   | AltiVec                                    | `ghash_vperm`      | 2.12.0     |
++-----------+--------------------------------------------+--------------------+------------+
+| ChaCha    | AltiVec                                    | `chacha_simd32`    | 2.8.0      |
++-----------+--------------------------------------------+--------------------+------------+
+| DARN      | POWER9                                     | `processor_rng`    | 2.15.0     |
++-----------+--------------------------------------------+--------------------+------------+
+| Serpent   | AltiVec                                    | `serpent_simd`     | 1.9.2      |
++-----------+--------------------------------------------+--------------------+------------+
+| SHACAL2   | AltiVec                                    | `shacal2_simd`     | 2.3.0      |
++-----------+--------------------------------------------+--------------------+------------+
+| NOEKEON   | AltiVec                                    | `noekeon_simd`     | 1.9.4      |
++-----------+--------------------------------------------+--------------------+------------+
+
+Configuring Acceleration
+------------------------------
+
+Hardware acceleration can be disabled at *compile time* by passing certain
+``--disable-*`` options to ``configure.py``. This will cause the base software
+implementation to be used instead of the hardware accelerated one.
+
```

Comment:
Might be worth mentioning here that `--disable-modules=` can be used to remove a specific feature, if this is desirable.

Reply:
Added that, thanks!